src/Solver/Local/Riemann/Roe.H

```diff
@@ -40,14 +41,20 @@ public:
     }
 
     int verbose = 0;
+    bool entropy_fix = false;
 
     static void Parse(Roe & value, IO::ParmParse & pp)
     {
         // enable to dump diagnostic data if the roe solver fails
-        pp.query_default("verbose",value.verbose,1);
+        pp.query_default("verbose", value.verbose, 1);
+        // apply entropy fix if tru
+        pp.query_default("entropy_fix", value.entropy_fix, false);
+
+        if (value.entropy_fix)
+            Util::Warning(INFO,"The entropy fix is experimental and should be used with caution");
```

Comment:
@etsandall I put the warning up here, so it only shows up once.
It's not implemented yet, but we can (and should) add warning tracking so the code exits with a warning that gets passed to the regtest system. 
For now, I think it's okay as-is.